.gitignore

```diff
@@ -31,3 +31,6 @@ coverage
 
 # Ignore ruby gemset files
 .ruby-gemset
+
+# Ignore OSX specific artifacts
```

Comment:
These kind of ignore files should normally be in our personal [git global ignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), they shouldn't be necessary to be added here.